src/components/pages/Home.tsx

```diff
@@ -29,7 +29,7 @@ const queryLatest = {
   query: {
     nativeSearch: 1,
     query_string: {
-      query: `-isInPurgatory:true`
+      query: `(service.attributes.main.type:dataset) -isInPurgatory:true`
```

Comment:
this slipped through, the _Recently Published_ section should show both so this should be enough:

```
query: `-isInPurgatory:true`
```

Reply:
good catch

Reply:
Totally forgot about the query. Thanks!